doc/crypto/api.db/psa/crypto.h

```diff
@@ -375,6 +380,9 @@ typedef struct psa_custom_key_parameters_t {
     /* implementation-defined value */
 #define PSA_TLS12_ECJPAKE_TO_PMS_OUTPUT_SIZE 32
 #define PSA_TLS12_PSK_TO_MS_PSK_MAX_SIZE /* implementation-defined value */
+#define PSA_WRAP_KEY_OUTPUT_SIZE(wrap_key_type, alg, key_type, key_bits) \
+    /* implementation-defined value */
+#define PSA_WRAP_KEY_PAIR_MAX_SIZE /* implementation-defined value */
```

Comment:
I would expect `PSA_WRAP_KEY_OUTPUT_MAX_SIZE` instead of `PSA_WRAP_KEY_PAIR_MAX_SIZE`.

Reply:
`psa_wrap_key()` has the same issue as `psa_export_key()` in relation to a MAX_SIZE macro for the output. There is no really helpful limit on the size of what can be stored in the keystore, for example certificates *could* be stored as `PSA_KEY_TYPE_RAW_DATA`. The description for the output buffer size states:

>* This API defines no maximum size for wrapped symmetric keys. Arbitrarily large data items can be stored in the key store, for example certificates that correspond to a stored private key or input material for key derivation.

For key export, we have `PSA_EXPORT_KEY_PAIR_MAX_SIZE`, `PSA_EXPORT_PUBLIC_KEY_MAX_SIZE` and (new for 1.3) `PSA_EXPORT_ASYMMETRIC_KEY_MAX_SIZE` (which is just the maximum of the other two). The reason for separate key-pair and public-key sizes is that these can be very different values, and most use cases would know which of these is required, permitting a more efficient choice of buffer size.

For key wrapping, I didn't include a macro for wrapped public keys, or asymmetric keys in general as I expected that there is no real use case for simple wrapping of public keys. If this assumption is wrong, I would be happy to add the additional macros to the API, or we might determine that a single one for wrapped asymmetric key maximum size is sufficient.

Reply:
Agreed, there is no general maximum size for export and key_wrap, thanks for the explanation!